### DISCUSSION TITLE
Porcessing Steps ? e.g Single image

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#224] [Q&A] [answered]
In the docu i found this 
Picture-Pipeline

"The single image pipeline runs following stages in given sequence:

    remove chromakey (greenscreen removal), replace by transparency
    apply pilgram2 filter (instagram like filters)
    fill background with a solid color
    add background image (useful only if no solid color was added)
    add a frame with transparent areas. Captured images shine through transparent area. Use PNGs with transparency!
    text overlay"

At what point do resizze the captrued image ?

My original has 1 MB
      unprocessed full 300 KB 
      processed full 3.4 KB with frame.

I'm asking because I'll try to speed up the processing and find the optimum sizze for frames and backgroung.

My gues is original is reduced to unprocessed full than the steps you descriped in the docu take place.

How do the steps corespond to the config parameters in the config ?
Mediaprocessing 
       Hires
       Fullstill

**Comment by mgineer85** [comment, marked as answer]
If you add a frame, that will be the output resolution. If you do not, the image of camera is the output resolution. 
maybe it’s not the best solution because coordinates change depending on the stage of frame enable or not.
